airflow/www/templates/airflow/dag.html

```diff
@@ -322,21 +322,28 @@ <h4 class="modal-title" id="dagModalLabel">
           </h4>
         </div>
         <div class="modal-body">
-          <form method="POST">
-            <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
-            <input type="hidden" name="dag_id" value="{{ dag.dag_id }}">
-            <input type="hidden" name="execution_date">
-            <input type="hidden" name="origin" value="{{ request.base_url }}">
-            <button type="button" id="btn_dagrun_clear" class="btn btn-primary" data-action="{{ url_for('Airflow.dagrun_clear') }}">
-              Clear
-            </button>
-            <button type="button" id="btn_dagrun_failed" class="btn btn-primary" data-action="{{ url_for('Airflow.dagrun_failed') }}">
-              Mark Failed
-            </button>
-            <button type="button" id="btn_dagrun_success" class="btn btn-primary" data-action="{{ url_for('Airflow.dagrun_success') }}">
-              Mark Success
-            </button>
-          </form>
+          <div class="row">
+            <span class="btn-group col-md-8">
+              <form method="POST">
+                <input type="hidden" name="csrf_token" value="{{ csrf_token() }}">
+                <input type="hidden" name="dag_id" value="{{ dag.dag_id }}">
+                <input type="hidden" name="execution_date">
+                <input type="hidden" name="origin" value="{{ request.base_url }}">
+                <button type="button" id="btn_dagrun_clear" class="btn btn-primary" data-action="{{ url_for('Airflow.dagrun_clear') }}">
+                  Clear
+                </button>
+                <button type="button" id="btn_dagrun_failed" class="btn btn-primary" data-action="{{ url_for('Airflow.dagrun_failed') }}">
+                  Mark Failed
+                </button>
+                <button type="button" id="btn_dagrun_success" class="btn btn-primary" data-action="{{ url_for('Airflow.dagrun_success') }}">
+                  Mark Success
+                </button>
+              </form>
+            </span>
+            <span class="col-md-4">
```

Comment:
This will align the button fully to the right.
```suggestion
            <span class="col-md-4 text-right">
```